server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java

```diff
@@ -26,6 +26,8 @@
 
     Mono<BlobId> save(byte[] data);
 
+    Mono<BlobId> save(String data);
```

Comment:
it could be declared as a default method

Reply:
hm I hesitated with that but got the remark once or twice that it was sometimes better to not declare a default method to enforce and make sure we don't forget it in some implementations if the behavior needs to be different. What are the Mokas thinking regarding this?

Reply:
nothing against default methods when appropriate